.chloggen/TEMPLATE.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type:
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component:
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: []
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+
+# If your change doesn't affect end users or the exported elements of any package,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+# Optional: The change log in which this entry should be included. In the semconv repo
```

Comment:
```suggestion
# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.

# Optional: The change log in which this entry should be included. In the semconv repo
```
You could also remove the chore/skip instruction to the top - it looks a bit out of place here.

Reply:
Good point. I moved it to the top 🔝